examples/qml_features/rust/src/nested_qobjects.rs

```diff
@@ -72,10 +69,11 @@ impl Default for OuterObjectRust {
 
 impl qobject::OuterObject {
     /// Print the count of the given inner QObject
-    //
-    // This method needs to be unsafe otherwise clippy complains that the
-    // public method might dereference the raw pointer.
-    unsafe fn print_count(self: Pin<&mut Self>, inner: *mut qobject::InnerObject) {
+    ///
+    /// # Safety
+    ///
+    /// This methods take a pointer so should consider safety when using the pointer
+    pub unsafe fn print_count(self: Pin<&mut Self>, inner: *mut qobject::InnerObject) {
         if let Some(inner) = unsafe { inner.as_ref() } {
```

Comment:
This `unsafe` block can go now, as the whole method is marked `unsafe`.

Reply:
Same for the `self.called` below.

Reply:
done